src/StepManager.cpp

```diff
@@ -114,12 +115,34 @@ MatchResult StepManager::stepMatches(const std::string &stepDescription) {
         const boost::shared_ptr<const StepInfo>& stepInfo = iter->second;
         SingleStepMatch currentMatch = stepInfo->matches(stepDescription);
         if (currentMatch) {
+            for (auto& submatch : currentMatch.submatches) {
```

Comment:
@src-ableton `cucumber-cpp` still supports C++98, you can't use for-range loops.

Reply:
cf https://travis-ci.org/AbletonAppDev/cucumber-cpp/jobs/545210181

Reply:
No longer using any syntax from C++ >0x.